docs/guides/lustre-tuning.md

```diff
@@ -0,0 +1,32 @@
+# Lustre Tuning
+`/capstor/` and `/iopsstor` are both [lustre](https://lustre.org) filesystem.
+Lustre is an open-source, parallel file system used in HPC systems.
```

Comment:
I think this sentence could be left out for brevity. It doesn't add much to the specific goal of teaching users about tuning the filesystem IMO.